e2e/testcases/resource_conditions_test.go

```diff
@@ -92,16 +92,29 @@ func TestResourceConditionAnnotations(t *testing.T) {
 	nt.MustKubectl("annotate", "configmap", cmName, "-n", ns,
 		`configmanagement.gke.io/errors=["CrashLoopBackOff"]`)
 
+	support, err := nt.SupportV1Beta1CRDAndRBAC()
```

Comment:
I'm wondering if we should just support one version and skip the test otherwise. Supporting both adds complexity to the assertions which I expect will confuse future people. WDYT?

Reply:
We can drop the support for v1beta1 version when we don't need to support k8s 1.22-. Currently, Config Sync supports k8s 1.20 to 1.24.